typescript/core/src/parse.ts

```diff
@@ -107,10 +112,9 @@ export function parseRecord({
         throw new Error(`Schema data length ${dataLen} exceeds bounds of record`);
       }
       const data = new Uint8Array(
-        recordView.buffer.slice(
```

Comment:
We've lost this copy - intentional?

Reply:
I restored this, benchmark data updated.